app/features/welcome/styled/Form.js

```diff
@@ -3,6 +3,6 @@
 import styled from 'styled-components';
 
 export default styled.form`
-    width: 350px;
-    margin: 0 15px;
+    width: 26em;
```

Comment:
Can you please make these changes a separate commit?

Reply:
Sure. I'll separate them after we finalize everything else.